app/cli/cmd/policy_develop.go

```diff
@@ -0,0 +1,31 @@
+//
+// Copyright 2025 The Chainloop Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"github.com/spf13/cobra"
+)
+
+func newPolicyDevelopCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "develop",
+		Aliases: []string{"dev"},
```

Comment:
```suggestion
		Aliases: []string{"devel"},
```

this sounds a little bit better to me since it might represent a verb